plugins/texteditor.koplugin/main.lua

```diff
@@ -28,6 +28,7 @@ local TextEditor = WidgetContainer:new{
     history_keep_size = 60, -- hom many to keep in settings
     normal_font = "x_smallinfofont",
     monospace_font = "infont",
+    default_font_size = 20, -- x_smallinfofont default size
```

Comment:
Should we pull that from Font.sizemap.x_smallinfofont or something?

Reply:
Yes, we could have. For next time then :)
(Although all these names don't really mean much when you read it... I usually select a font size that pleases me, and pick a name that happens to have that same size :)